apps/sms/js/subject_composer.js

```diff
@@ -62,6 +62,13 @@
     }
   }
 
+  function updateDirection(value) {
+    /* jshint validthis: true */
+    var priv = privateMembers.get(this);
+
+    priv.input.dir = !value ? '': 'auto';
+  }
```

Comment:
Since the caret will be place to left easily once you clear the content, I feel it would be created by QA someday in the future...